utils/utils.js

```diff
@@ -93,3 +93,13 @@ module.exports.translateWhereToQuery = function (where, str, key) {
   }
   return str
 }
+
+/**
+ * Gets the right url from the config
+ * @param {string} url
+ * @returns {string}
+ */
+module.exports.getMainUrl = function (url) {
+  const mainUrl = url.split('/').pop() === '$metadata' ? url.slice(0, -9) : url
```

Comment:
Instead of `url.slice(0, -9)` you can use `url.replace('$metadata', '')`, just because it's most common practice.